submissions/dyworld-history/dyworld-history.md

```diff
@@ -0,0 +1,49 @@
+
+The DyTech was very populair back in the day. I even played together a few times with Xterminator and Oninaig while they were doing a lets play on Youtube. After a few years (Factorio 0.15) I started to work on the next step of DyTech, DyWorld. A major rewrite after 2 years of no modding, to fit DyTech better into the ever evolving Factorio. I made great strides, and worked on it till just before Factorio 1.0.0. Several content creators played it on Youtube:
+[KatherineOfSky DyWorld Playthrough](https://www.youtube.com/playlist?list=PL4o6UvJIdPNqubR5oXdx9SqKFoYW_SL-q), 
+[Nexxus Gaming DyWorld Playthrough](https://www.youtube.com/watch?v=Eqs8dakCzL8&list=PLXEOcXLt0r4RxjOgak-BRzO8PMoMnqnoy), 
+[Lumatan DyWorld Playthrough (German)](https://www.youtube.com/watch?v=eH6lu5N-oPQ&list=PLEui1S1GUHOMKnVrLjgybKRKzBNMIHTfk)
+
+About august of last year my itch to mod was back, bigger then before. I decided to rewrite the mod, and add the content of the mod around 1 very specific thing in mind: A story. Thus I rewrote DyWorld into DyWorld-Dynamics.
+But it is now almost a year later, and it is still not released. Well, due to some personal issues I could not do anything a few months. When that was taken care of, I returned to keep working on it. And now it is getting closer to release. Adding in content around a story is much more work then I previously anticipated, but it is fun to do. Luckily I have several beta testers to help me take care of issues while I write everything.
+
+## The Story ##
+
+This is my main focus of DyWorld-Dynamics. I wrote everything with the story elements in mind, but it will work without it as well. What is the story about?
+
+The story starts with crash landing on a desolate planet, filled with weird flora and fauna. You have no idea how you got there and where you are. As the story progresses, you name the planet, find out why you crash landed there, learn about other planets and move there and find an epic conclussion of the story.
+
+This is what will be available once it publicly releases. 3 Acts of story, with multiple objectives. the first 2 acts will learn you the basic mechanics of DyWorld-Dynamics with its complex recipes, and act 3 will completly focus on the story element that you can check out on your own pace
+
+The mod does require a few other mods though, most notable are Nanobots and Space Exploration, since they are now required for the story. Most overhaul mods will be incompatible. Smaller mods will be compatible, but this requires testing, and you can ask for that on my [Discord](https://discord.gg/yR5vBWy).
```

Comment:
I don't think this paragraph belongs here, or is really neccesary, and seems a little like an after thought. I would just work Space Exploration into the previous paragraph. Something like

> This is what will be available once it publicly releases. 3 Acts of story, with multiple objectives, and travelling between planets powered by the beautiful Space Exploration mod. The first two acts...